DEVELOPMENT.md

```diff
@@ -130,3 +130,19 @@ $ bin/yard server --reload
 
 Then navigate to `localhost:8808` to view the rendered docs.
 
+### Publish documentation to rspec.github.io
+
+In other documented rspec gems, we use a rake command from rspec-dev
+to generate documentation. `rspec-rails` is no more synced with
+other gems in term of version. If you want to update the documentation
+to [`rspec.github.io`](https://github.com/rspec/rspec.github.io) you
+need to git clone in the parent folder of rspec-rails the `rspec.github.io`
+repository. You also need from [source branch](https://github.com/rspec/rspec.github.io/tree/source)
+the gem that will be needed as yard plugin: [rspec-docs-template](https://github.com/rspec/rspec.github.io#install-rspec-docs-template-yard-plugin).
```

Comment:
Maybe the commands are not needed. We just need to refer to other repo?

Reply:
I'd prefer only commands 😆